src/plugins/core/patches/commands/insert-list.js

```diff
@@ -1,4 +1,4 @@
-define([], function () {
```

Comment:
There might have been an obscure module bug that meant we kept the empty arrays. Could you submit this change in a different PR so that we don't have yank the consolidation because of the loader.